code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -161,6 +161,20 @@
 /datum/dynamic_ruleset/midround/from_ghosts/proc/setup_role(datum/antagonist/new_role)
 	return
 
+/// Fired when there are no valid candidates. Will spawn a sleeper agent or latejoin traitor.
+/datum/dynamic_ruleset/midround/from_ghosts/proc/attempt_replacement()
+	var/datum/dynamic_ruleset/midround/autotraitor/sleeper_agent = new
+
+	// Otherwise, it has a chance to fail. We don't want that, since this is already pretty unlikely.
+	sleeper_agent.randomly_acceptable = FALSE
```

Comment:
Not sure if I like this variable name.

This is a variable which bypasses the normal prob chance that this ruleset will fail.

Maybe `bypass_failure_chance`?

Reply:
Sure, I just don't like inverted variable names.

Reply:
`has_failure_chance`?

Reply:
thats good yeah, ill do it later